base/rational.jl

```diff
@@ -4,7 +4,27 @@
     Rational{T<:Integer} <: Real
 
 Rational number type, with numerator and denominator of type `T`.
-Rationals are checked for overflow.
+
+The preferred way of creating `Rational` numbers is using the
+`//` operator. `Rational`s are also checked for overflow.
```

Comment:
This seems unnecessarily opinionated to me. Further, I don't think `//` is always the best way. For example, sometimes it is better to use `rationalize`.